pkg/internal/ebpf/common/pids.go

```diff
@@ -117,6 +137,10 @@ func (pf *IdentityPidsFilter) AllowPID(_ uint32) {}
 
 func (pf *IdentityPidsFilter) BlockPID(_ uint32) {}
 
+func (pf *IdentityPidsFilter) CurrentPIDs() map[uint32]struct{} {
```

Comment:
I added this function so I can grab the list of current active pids when we need it. I'll explain in httpfltr below.